website/docs/docs/collaborate/data-tile.md

```diff
@@ -0,0 +1,211 @@
+
+5. Copy the **Metadata Only token** and save it in a secure location. You'll need it token in the next steps.
+6. Navigate back to dbt Explorer and select an exposure.
+7. Below the **Data health** section, expand on the toggle for instructions on to embed the exposure tile (if you're an account admin with develop permissions). 
+8. In the expanded toggle, you'll see a text field where you can paste your **Metadata Only token**.
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-tile-example.jpg" width="85%" title="Expand the toggle to embded data health tile into your dashboard." />
+
+9. Once you’ve pasted your token, you can select either **URL** or **iFrame** depending on which you need to install into your dashboard.
+
+If your analytics tool supports iFrames, you can embed the dashboard tile within it. 
+
+### Embed data health tile in Tableau
+To embed the data health tile in Tableau, follow these steps:
+
+1. Ensure you've copied the embed iFrame content in dbt Explorer.
+2. For the revamped environment-based exposure tile you can insert these fields into the following iFrame, and then embed them with your dashboard. This is the iFrame that is available from the **Exposure details** page in dbt Explorer.
+
+    `<iframe src='https://metadata.YOUR_ACCESS_URL/exposure-tile?uniqueId=<exposure_unique_id>&environmentType=production&environmentId=<environment_id>&token=<metadata_token>' />`
+
+    *Note, replace the placeholders with your actual values.*
+
+<DocCarousel slidesPerView={1}>
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-tile-iframe.jpg" width="70%" title="Example of embedded iFrame" />
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-tile-pass.jpg" width="60%" title="Example of passing Data health tile in your dashboard." />
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-tile-stale.jpg" width="60%" title="Example of stale of degraded Data health tile in your dashboard." />
+</DocCarousel>
+
+
+3. For the job-based exposure tile you can insert these three fields into the following iFrame, and then embed them with your dashboard. The next section will have more details on the job-based exposure tile.
+
+    `<iframe src='https://metadata.YOUR_ACCESS_URL/exposure-tile?name=<exposure_name>&environment_id=<environment_id>&token=<metadata_token>' />`
+
+    *Note, replace the placeholders with your actual values.*
+
+## Job-based data health <Lifecycle status="Legacy"/>
+
+The default experience is the [environment-based data health tile](#view-exposure-in-dbt-explorer) with dbt Explorer.
+
+This section is for legacy job-based data health tiles. If you're using the revamped environment-based exposure tile, refer to the previous section. Expand the following to learn more about the legacy job-based data health tile. However, note that 
```

Comment:
Hiya @mirnawong1 

Just a QQ on this, is the last sentence "However, note that" incomplete?

Kind Regards